### ISSUE TITLE
Nametags incorect render

### ISSUE THREAD

**Issue by Nekiplay** [MeteorDevelopment/meteor-client#5042]
### Describe the bug

Incorect rendering items in nametags
![image](https://github.com/user-attachments/assets/7b5dffe3-d855-4928-92a3-33002ff87d82)

Incorect rendering items in holograms in Meteor+
![image](https://github.com/user-attachments/assets/65cbf279-05f2-4d9f-a37f-71be2886d4e9)

### Steps to reproduce

Mods:
![image](https://github.com/user-attachments/assets/31c0d52d-5623-4e47-a3da-5b8681119c5f)

### Meteor Version

0.5.9-2213

### Minecraft Version

1.21.3

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Pathfinder455** [comment]
Seconded
Mod list Baritone, Meteor client, Meteor+, Sodium, Iris

**Comment by Nekiplay** [comment]
I also tested without third-party mods problem also occurs

**Comment by Nekiplay** [comment]
Checked without third-party mods bug is also present
![image](https://github.com/user-attachments/assets/a870bf0c-7c62-464e-87bc-25df9f39072d)

**Comment by Banana1059** [comment]
Can at least confirm it happens in 1.20.4 because I've had it happen to me while on it

**Comment by Nekiplay** [comment]
> Can at least confirm it happens in 1.20.4 because I've had it happen to me while on it

I have everything working fine on 1.20.4

**Comment by bebapainen** [comment]
having this too. 
mc 1.21.3
windows 11

modlist

baritone-1.21.3-20241117.084726-1.jar
meteor-client-0.5.9-2213.jar
ViaFabricPlus-3.5.3.jar

**Comment by chairclr** [comment]
This is caused by players wearing unusually shaped heads, such as Ender Dragon or Piglin heads

**Comment by Nekiplay** [comment]
> This is caused by players wearing unusually shaped heads, such as Ender Dragon or Piglin heads

No, this is caused by an error in the RenderUtils.drawItem method
![image](https://github.com/user-attachments/assets/d3f215d5-af59-4920-881a-fc00a886e5fa)

**Comment by MineGame159** [comment]
Fixed in f33a21b174f812f04852ff3328187233a00f9316.
